tests/flows/steps/test_create_new_aws_environment.py

```
@@ -0,0 +1,203 @@
+            master_payer_id=get_mpa_account_id(context.order),
+        ),
+        title=CRM_NEW_ACCOUNT_TITLE,
+    )
+    mock_crm_client.create_service_request.assert_called_once_with(
+        context.order_id, expected_service_request
+    )
+    assert get_crm_new_account_ticket_id(context.order) == "TICKET-123"
+
+
+def test_process_raises_query_error_mpa_missing(
+    mocker,
```

Ruff (0.14.10)

Unused function argument: `mocker`

(ARG001)